roles/openshift_node_upgrade/tasks/restart.yml

```diff
@@ -26,12 +26,30 @@
   openshift_facts:
     role: docker
 
+- name: Create ovsdb-server systemd dropin folder
```

Comment:
This would be best moved to a separate task file and included here, so that `when: openshift_use_openshift_sdn` would not have to be duplicated

Reply:
Thanks for the suggestion.I implemented the change. Let me know how it looks.